src/lib/zork/zork.ts

```diff
@@ -359,36 +352,35 @@ const zorkServer = net.createServer((client) => {
   });
 
   client.on('end', () => {
-    console.info(`client ${ctx.clientId} disconnected`);
-    if (ctx.mode === 'give') {
-      npeersGetting--;
-      console.info('decremented npeersGetting to', npeersGetting);
+    ctx.log(`[zorkServer] client disconnected`);
+    if (ctx.mode === 'give' && ctx.rtc.conn) {
+      numGetters--;
+      ctx.log(`decremented numGetters to ${numGetters}`);
     }
```

Comment:
Oh, I didn't realize that! I'll back out a0e7248 and change the logic to conform with this.